src/Proto/nodeguard.proto

```diff
@@ -240,3 +243,21 @@ message GetChannelOperationRequestByIdResponse {
   bool private = 12;
   string job_id = 13;
 }
+
+message AddLiquidityRuleRequest {
+// Channel ID as returned by the Lightning implementation
+  int32 channel_id = 1;
```

Comment:
I think SCIDs (short channel ids) are `uint64`

Reply:
https://github.com/lightningnetwork/lnd/blob/d24f12bd0bbc704560c1092a44fc9883a0ef30f8/lnrpc/invoicesrpc/addinvoice.go#L578

Reply:
Wrong explanation comment. This is a reference of the ID of the channel in NGs database

Reply:
NIT: Then maybe you could have edited the `rule` var directly?